content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,197 @@
+
+### MeshRootCertificate
+
+Each `MeshRootCertificate` represents a certificate that can be used in the Mesh. At most there can be only two `MeshRootCertificate`'s that are used in the mesh at a given time and there must always be one `MeshRootCertificate` in the **active** `role`.
+
+The following are the validate values for a `MeshRootCertificate` `role`:
+
+- `active`: The certificate is used to sign **and** validate certificates in the mesh. There must always be one certificate with active.
+- `passive`: The certificate is used to validate certificates in the mesh
+- `inactive`: The certificate is not used in the mesh.
+
+The following is an example of the `MeshRootCertificate` that will be created. The Provider option can be `tresor`, `certManager`, or `vault`.  Before using the providers for [CertManager](certificates.md#using-cert-manager) or [Vault](certificates.md#using-hashicorp-vault), additional pre-configuration steps are required.  Refer to the [API documentation](#TODO) for how to configure the `MeshRootCertificate` fields for each provider.
+
+Run `kubectl get meshrootcertificate` to see the default `MeshRootCertificate`:
+
+```yaml
+apiVersion: config.openservicemesh.io/v1alpha2
+kind: MeshRootCertificate
+metadata:
+  name: osm-mesh-root-certificate
+  namespace: osm-system
+spec:
+  role: active
+  provider:
+    tresor:
+      ca:
+        secretRef:
+          name: osm-ca-bundle
+          namespace: osm-system
+  spiffeEnabled: false
+  trustDomain: cluster.local
+```
+
+The only field that can be modified after initial creation is the `role` field.  Other fields will need change by creating a new MeshRootCertificate and go through the [rotation process](#meshrootcertificate-rotation). On a new install, the initial `MeshRootCertificate` will be created by OSM and the certificate will be in the **active** `role`. Fields such as `trustDomain` and `spiffeEnabled` can be configured by passing options to the install command:
+
+```
+osm install --set="osm.featureFlags.enableSPIFFE=true --set"osm.trustDomain=cluster.example"
+```
+
+### MeshRootCertificate Rotation
+
+To rotate the root certificate a mesh, a second `MeshRootCertificate` (MRC) will be created.  The second `MeshRootCertificate` will be created in a `passive` role and rotated into the `active` role. Once the new certificate is `active`, the old certificate will be moved to the `passive` role then finally `inactive`.  
+
+The full process for rotating the certificate is:
+
+| Step  |  original MRC(1) role   |  new MRC(2) role    | Signing MRC  | Validating MRC |
```

Comment:
There are some parts of the table that uses `mcr` instead of `mrc`

Reply:
fixed!